dependencyversions.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For consistency I would call this dependencies.props. It also looks like you have configured the VersionTools stuff for updating yet.